native/tools/swig-config/tutorial/index.md

```diff
@@ -689,6 +711,95 @@ Build and run project
 18:09:53 [DEBUG]: D/ JS: ==> new: myObj.type: 888 // Cool, C++ getType is invoked, 888 is return from C++
 ```
 
+#### %attribute_writeonly  directive
+
+`%attribute_writeonly` directive is an extension we added in swig  `Cocos` backend, it's used for the purpose that C++ class only has a `set` function and there isn't a `get` function.
+
+In `native/tools/swig-config/cocos.i`, there are:
+
+```c++
+%attribute_writeonly(cc::ICanvasRenderingContext2D, float, width, setWidth);
+%attribute_writeonly(cc::ICanvasRenderingContext2D, float, height, setHeight);
+%attribute_writeonly(cc::ICanvasRenderingContext2D, float, lineWidth, setLineWidth);
+%attribute_writeonly(cc::ICanvasRenderingContext2D, ccstd::string&, fillStyle, setFillStyle);
+%attribute_writeonly(cc::ICanvasRenderingContext2D, ccstd::string&, font, setFont);
+```
+
+This is the similar functionality like which in JS:
+
+```javascript
+Object.defineProperty(MyNewClass.prototype, 'width', {
+  configurable: true,
+  enumerable: true,
+  set(v) {
+    this._width = v;
+  },
+	// No get() for property
+});
+```
+
+#### Reference type
+
+If C++ `get` and `set` function return reference type, don't forget to add `&` suffix in %attribute or %attribute_writeonly directives. The following `ccstd::string&` is an example.
+
+```c++
+%attribute_writeonly(cc::ICanvasRenderingContext2D, ccstd::string&, fillStyle, setFillStyle);
+```
+
+If `&` is missing, temporary `ccstd::string` objects will be created while binding functions are invoked.
```

Comment:
- temporary `ccstd::string` objects -> a temporary `ccstd::string` instance?
- while binding functions are invoked -> while the binding function is invoked

Reply:
DONE